turn_alerts/utils.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, division
+
+import base64
+import hmac
+from hashlib import sha256
+
+from rest_framework.exceptions import AuthenticationFailed
+
+from .models import Organization, TurnSecret
+
+
+def validate_signature(request, org_id, turn_secret_id):
+    try:
+        organization = Organization.objects.get(id=org_id)
```

Comment:
This will mean we're doing the org lookup twice, remove this and replace the line under it:
`turn_secret = TurnSecret.objects.get(id=turn_secret_id, org_id=org_id)`

and remove the org exception